### PR TITLE
Display missing resource dialog not less than 5 seconds

### PR DESCRIPTION
**Pull request by ihhub**
In this case users won't claim that the application crashes for no reason.

### CODE REVIEW

File: src/fheroes2/game/fheroes2.cpp [hunk 1]
```diff
@@ -128,8 +128,20 @@ namespace
         display.render();
 
         LocalEvent & le = LocalEvent::Get();
-        while ( le.HandleEvents() && !le.KeyPress() && !le.MouseClickLeft() ) {
-            // Do nothing.
+
+        // Display the message for 5 seconds so that the user sees it enough and not immediately closes without reading properly.
+        fheroes2::Time timer;
```
[the remark this answers is not in the document]
Comment (ihhub) [reply src/fheroes2/game/fheroes2.cpp:133]: ... yes :)

**Review by oleg-derevenetz — CHANGES_REQUESTED** [review]
Hi @ihhub please calm down the IWYU and add the header for `fheroes2::Time` :)

**Comment by ihhub** [comment]
@oleg-derevenetz , IWYU is my bigger enemy in PRs :)

**Review by oleg-derevenetz — APPROVED** [review]

**Review by Districh-ru — APPROVED** [review]
👍

**Pull request merged**